schema.sql

```diff
@@ -100,6 +100,12 @@ role_capability.capability_id = capability.capability_id JOIN role ON role_capab
 ON band.band_id = role.band_id 
 ORDER BY band.band_id ASC, capability.capability_id ASC;
 
+CREATE VIEW viewBandData AS
+SELECT band.band_id, band.name, band.school_id, band.description_id,commercial_awareness, communicating_and_teamwork, innovation_and_continuous_improvement,
+customer_focus, developing_yourself_and_others, planning_and_organising, job_specific_knowledge
+FROM band JOIN  band_description_id ON band.description_id = band_description_id.description_id;
+
+
```

Comment:
On actual projects, you'll notice that removing whitespace is a common issue raised in pull requests. I suggest removing this line to keep formatting consistent and tidy across the whole file.